pkg/limits/rate_limiting.go

```diff
@@ -199,6 +201,12 @@ var configs = []counterConfig{
 		Limit:  30,
 		Period: 1 * time.Hour,
 	},
+	// JobThumbnail
```

Comment:
Probably this comment is a copy/paste junk

Reply:
Yes, good catch!